testdata/discover-releases-scenario-2-stable-4.16.json

```diff
@@ -0,0 +1,11 @@
+{
+  "nodes": [
+    {
+      "version": "4.16.2",
+      "payload": "payload-stable",
+      "metadata": {
+        "io.openshift.upgrades.graph.release.channels": "stable-4.16, fast-4.16"
```

Comment:
I haven't gone to check what you're trying to test, but in production, there are no spaces after the comma:

```console
$ curl -s 'https://api.openshift.com/api/upgrades_info/graph?channel=stable-4.16' | jq -r '.nodes[] | select(.version == "4.16.2").metadata["io.openshift.upgrades.graph.release.channels"]'
candidate-4.16,eus-4.16,fast-4.16,stable-4.16,candidate-4.17,fast-4.17,stable-4.17,candidate-4.18,fast-4.18
```

Reply:
good point, i will adjust the input data then. i am testing if versions can be discovered dynamically by discovering the new channels stored in the metadata.